src/index.js

```diff
@@ -134,8 +138,12 @@ class Dropzone extends React.Component {
   }
 
   open() {
+    const clickEvent = new MouseEvent('click', {
+      bubbles: false
+    });
+
     this.fileInputEl.value = null;
-    this.fileInputEl.click();
+    this.fileInputEl.dispatchEvent(clickEvent);
```

Comment:
You mean like adding `onClick: (e) => { e.stopPropagation(); }` here? https://github.com/okonet/react-dropzone/blob/master/src/index.js#L207-L214